qrenderdoc/Windows/PipelineState/VulkanPipelineStateViewer.cpp

```diff
@@ -41,6 +41,8 @@ Q_DECLARE_METATYPE(CombinedSamplerData);
 
 namespace
 {
+constexpr uint32_t VK_ATTACHMENT_UNUSED = ~0U;
```

Comment:
I'd rather make this an `UnusedLocation` constant in `VKPipe::RenderPass` so that it's explicitly documented and the magic value is surfaced for python users as well.

Reply:
Ok, this is a Vulkan constant, I renamed it (not to conflict with Vulkan's `#define`) and put it there. Is `static constexpr` ok, or should this be a normal member for it to work in Python? I went with normal for now, but `static constexpr` would make more sense.